src/ActionHandler/SynchronizeCollectionHandler.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace CommonGateway\VrijBRPToZGWBundle\ActionHandler;
+
+use App\ActionHandler\ActionHandlerInterface;
+use CommonGateway\VrijBRPToZGWBundle\Service\NewSynchronizationService;
+use CommonGateway\VrijBRPToZGWBundle\Service\VrijBrpService;
+
+class SynchronizeCollectionHandler implements ActionHandlerInterface
+{
+
+
+    /**
```

Comment:
Missing short description in doc comment

Reply:
Fixed in PR #4